### ISSUE TITLE
It removes the link to the first version

### ISSUE THREAD

**Issue by mitar** [oscarotero/keep-a-changelog#21]
The example changelog [here](https://keepachangelog.com/en/1.0.0/) links to the first version, too. The link is to:

```
[0.0.1]: https://github.com/olivierlacan/keep-a-changelog/releases/tag/v0.0.1
```

This package removes such link.

**Comment by oscarotero** [comment]
mmm, good point!
I'll work on that. Thanks!

**Comment by mitar** [comment]
Maybe add a test which takes the changelog from https://keepachangelog.com/en/1.0.0/ and makes sure it does not change. :-)

**Comment by oscarotero** [comment]
Fixed in v2.0.0

**Comment by diprokon** [comment]
Hi!

I had a similar issue. After long debugging, I found out, that my file was in `CRLF`, and after [`split('\n')`](https://github.com/oscarotero/keep-a-changelog/blob/9b6b96d8ad73ddb9e617b8e066dd1da821980f96/src/parser.ts#L154) every line had `\r` in the end. Because of it, [the link regexp](https://github.com/oscarotero/keep-a-changelog/blob/9b6b96d8ad73ddb9e617b8e066dd1da821980f96/src/parser.ts#L182) for parsing links didn't work

To fix it, I just change my file to `LF`

Hope my finding will be helpful
